### ISSUE TITLE
Gibberish text on Windows 10

### ISSUE THREAD

**Issue by ozac** [edwig/ODBCQueryTool#1]
Hi,

Running on Windows 10, the text (and connection name) appear as gibberish.
On Windows 2012 it works well.

Is there a way to fix that?

Thanks

![image1](https://user-images.githubusercontent.com/16306656/27011851-c22f3920-4ecd-11e7-8d2c-6640b4415b14.png)

**Comment by edwig** [comment]
Hi,

 

Very strange, as I am running and compiling this on Windows 10 myself, for all the databases it supports.

Can you provide me with any more details about the ODBC driver you are trying to connect to?

Was the text in the “Text2” pane newly typed text, or was it an existing file?

 

Thanks in advance!

 

Edwig Huisman

 

Van: ozac [mailto:notifications@github.com] 
Verzonden: zondag 11 juni 2017 16:50
Aan: edwig/ODBCQueryTool <ODBCQueryTool@noreply.github.com>
CC: Subscribed <subscribed@noreply.github.com>
Onderwerp: [edwig/ODBCQueryTool] Gibberish text on Windows 10 (#1)

 

Hi,

Running on Windows 10, the text (and connection name) appear as gibberish.
On Windows 2012 it works well.

Is there a way to fix that?

Thanks

 <https://user-images.githubusercontent.com/16306656/27011851-c22f3920-4ecd-11e7-8d2c-6640b4415b14.png>

**Comment by ozac** [comment]
Thanks for the quick response.

The text is actually the `>> EOF <<` line which appears at the end of each file.
If I try to type something, it displays as gibberish. Same thing if I open an existing text file.

I tried to change the font under Settings -> Fonts & Colors.
I also tried to run in compatibility mode (right-click -> properties -> Compatibility).
It still doesn't work.

Windows version is 10.0.15063.

Thanks

**Comment by edwig** [comment]
Hi Ozac,

 

Very strange indeed. Certainly after reviewing everything that the application does after opening a file buffer. It seems to me that your installation does ‘something’ with a Unicode buffer or file, while the application is strictly MBCS.

I would recommend the following:

1.	Deinstalling the application
2.	Make sure that “C:\Program Files\EDO\ODBCQuerytool\Data” directory is completely gone
3.	Download the last version from sourceforge (installer *.exe of version 2.2.0) at this location: https://sourceforge.net/projects/odbcquerytool/files/odbcquerytool/Release%202.2.0/
4.	Reinstall the program

☹

 

 

Van: ozac [mailto:notifications@github.com] 
Verzonden: zondag 11 juni 2017 17:42
Aan: edwig/ODBCQueryTool <ODBCQueryTool@noreply.github.com>
CC: Edwig Huisman <edwig.huisman@hetnet.nl>; Comment <comment@noreply.github.com>
Onderwerp: Re: [edwig/ODBCQueryTool] Gibberish text on Windows 10 (#1)

 

Thanks for the quick response.

The text is actually the >> EOF << line which appears at the end of each file.
If I try to type something, it displays as gibberish. Same thing if I open an existing text file.

I tried to change the font under Settings -> Fonts & Colors.
I also tried to run in compatibility mode (right-click -> properties -> Compatibility).
It still doesn't work.

Thanks
